lib/server.ml

```diff
@@ -37,6 +39,9 @@ let make channel =
   { channel; request; reply; m }
 
 let connect channel ?offer () =
+  let section = Lwt_log_core.Section.make("Server.connect") in
```

Comment:
`Lwt_log_core` is opened in `Lwt_log`, please use `Lwt_log` instead (this is used also later on)

Reply:
The reason I used `Lwt_log_core` in `lib/` instead of `Lwt_log` is to avoid using any unix-dependent things, like the syslog, that are in `Lwt_log` - `lib/` has to be compatible with Mirage.
The [documentation](http://ocsigen.org/lwt/2.7.1/api/Lwt_log) of `Lwt_log` says:

> This module provides functions to deal with logging. It extends Lwt_log_core with Unix features.

Reply:
Good point!